sos/report/plugins/docker.py

```diff
@@ -82,7 +82,7 @@ def setup(self):
         for container in containers:
             self.add_cmd_output("docker inspect %s" % container,
                                 subdir='containers')
-            if self.get_option('logs'):
+            if self.get_option('logs') or self.get_option("all_logs"):
```

Comment:
After some more thought and discussion on this with another maintainer, I don't think this is desirable for most users of sos. The plugin option serves as sufficient enough gating for collecting container logs, which are historically not needed to diagnose host issues.

If a user/consumer of sosreports is looking into containerized applications, then it's reasonable to assume they know to use `-k docker.logs`. Conversely, `--all-logs` is going to be used more often and in more use cases, in which getting (non-size-limited) container logs is likely going to be surprising behavior to users.

Reply:
ok, I've reverted the changes.